documentation/Changelog-for-next-generation.md

```diff
@@ -0,0 +1,8 @@
+Compared to the current management libraries, the next-generation libraries have the following changes:
```

Comment:
"current" is a relative time term and might cause misunderstanding, let's just say

The next-generation Azure JavaScript libraries introduces a few important changes: